data/assets/scene/solarsystem/planets/mercury/layers/colorlayers/messenger_mdis_sweden.asset

```diff
@@ -1,13 +1,34 @@
+local assetHelper = asset.require("util/asset_helper")
 local globeIdentifier = asset.require("../../mercury").Mercury.Identifier
 
 local layer = {
   Identifier = "Messenger_MDIS_Sweden",
   Name = "Messenger MDIS [Sweden]",
-  FilePath = asset.localResource("messenger_mdis_sweden.wms")
+  FilePath = asset.localResource("messenger_mdis_sweden.wms"),
+  Description = [[This May 2013 basemap is a combination of the following mosaics; (1) 
+  The 2013-05-10 version of the monochrome global mosaic, made from Applied Coherent 
+  Technology (ACT) Corporation tiles, (2) An average north polar mosaic from 90N to 82.5N, 
+  composed of images from many campaigns, made by C. Ernst, and (3) An average south 
+  polar mosaic from 90S to 85S, composed of images from the south polar monitoring 
+  campaign from the primary mission, made by N. Chabot. To fill minor areas of missing 
+  data, the 2013-05-10 version of the high-incidence global mosaic was underlain. This 
+  monochrome mosaic is composed of Mercury Dual Imaging System (MDIS) Narrow Angle 
+  Camera (NAC) images and Wide Angle Camers (WAC) images acquired in the filter centered 
+  at 750 nm. The resolution of this mosaic is 250 meters per pixel (m).]]
 }
 
-asset.onInitialize(function () 
-  openspace.globebrowsing.addLayer(globeIdentifier, "ColorLayers", layer)
-end)
+assetHelper.registerLayerAndExport(asset, layer, globeIdentifier, "ColorLayers")
+
+
+asset.meta = {
+  Name = "Messenger MDIS [Sweden]",
+  Version = "1.0",
+  Description = [[ Mercury MDIS layer for Mercury globe. 
```

Comment:
same here, Messenger?

Reply:
And for all the other layers that start with "Messenger"

But maybe actually, "Mercury Messenger XX layer"? As you did for MOsaic2, for example